docs/articles/improve-raptor-with-rag.md

```diff
@@ -0,0 +1,555 @@
+![Implementation of RAPTOR](../assets/use_cases/improve-raptor-with-rag/raptor-1.png)
+
+Traditional [RAG](https://vipul-maheshwari.github.io/2024/02/14/rag-application-with-langchain) setups often split documents into fixed-size chunks, but this can lead to problems in maintaining the semantic coherence of the text. If a key concept spans multiple chunks, and we only retrieve one chunk, the LLM might lack the full understanding of the idea, leading to incomplete or misleading responses. As a result, crucial ideas and relationships between concepts may be overlooked, leading to incomplete or inaccurate responses.
+
```

Comment:
I thought this was going to turn into a diagram. Can you explain what you're trying to illustrate with this image?

Reply:
Well the image meant nothing! I mean it;s just me implementing the RAPTOR :)